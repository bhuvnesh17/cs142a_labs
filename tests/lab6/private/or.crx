// or

func main() : void {
  	if (false or true) {
  	  	::printInt(101);
  	} else {
  		::printInt(10);
  	}
  	::println();
  	var a : bool;
  	var b : bool;
  	let a = true;
  	let b = true;
  	if (a or b) {
  	  	::printInt(101);
  	} else {
  		::printInt(10);
  	}
  	::println();
  	let a = false;
  	let b = true;
  	if (a or b) {
  	  	::printInt(101);
  	} else {
  		::printInt(10);
  	}
  	::println();
  	let a = true;
  	let b = false;
  	if (a or b) {
  	  	::printInt(101);
  	} else {
  		::printInt(10);
  	}
  	::println();
  	let a = false;
  	let b = false;
  	if (a or b) {
  	  	::printInt(101);
  	} else {
  		::printInt(10);
  	}
  	::println();
}

