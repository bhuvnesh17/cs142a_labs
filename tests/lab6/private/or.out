101
101
101
101
10
